tutorials/index.rst

```diff
@@ -11,6 +11,7 @@ find a tutorial of interest.
    :maxdepth: 2
    :caption: Tutorials
 
+   ../auto_examples/index
```

Comment:
nit: why don't we re-order the tutorials list since its becoming more generic for users and less lab specific.

I'll say we do ...

commands
python api examples
lab
integrations

Reply:
Yeo that's a good idea - what do you think about moving the entire tutorials block up too?

Reply:
i think that's a good idea, since it has more "beginner" stuff vs the Jobs section. 

I could see an argument that it goes above FAQ too .... but since the FAQ starts with a "what is flux?" question, I'd say we leave FAQ above Tutorials.  But that's just my opinion.

Reply:
Updated.